test/BidderRegistryTest.sol

```diff
@@ -135,7 +135,7 @@ contract BidderRegistryTest is Test {
         bidderRegistry.prepay{value: 2 ether}();
         address provider = vm.addr(4);
         bidderRegistry.LockBidFunds(bidID, 1 ether, bidder);
-        bidderRegistry.retrieveFunds(bidID, payable(provider));
+        bidderRegistry.retrieveFunds(bidID, payable(provider),100);
```

Comment:
Am I interpreting the `100` percent here correctly in that decay is disabled, and this file doesn't test the decay mechanism?

Reply:
Not this file, but this test doesn't test the decay. You're interpreeting it correctly. I've tested with decayed values in other tests. Wanted to isolate the behaviour to give better signal in the case where we make a breaking change.